pkg/dynamic_classifier/dynamic_classifier.go

```diff
@@ -224,6 +226,10 @@ func (dc *DynamicClassifier) loadMatchesFromCSV(matcher matcher, path string) er
 			return err
 		}
 
+		if len(line) != 4 {
+			dc.logger.WithField("line", line).WithField("file", path).Errorf("invalid CSV row format")
```

Comment:
The error message could be better, for instance `invalid CSV - different number of columns detected (expecting %s columns)`